lib/rack/show_exceptions.rb

```diff
@@ -93,7 +93,11 @@ def pretty(env, exception)
         end
       }.compact
 
-      TEMPLATE.result(binding)
+      template.result(binding)
+    end
+
+    def template
+      self.class::TEMPLATE
```

Comment:
Referencing `TEMPLATE` in this way allows subclasses to override the `TEMPLATE` constant. Subclasses also have the choice of overriding the `#template` method instead.

Reply:
Thanks. It is better to use just `TEMPLATE` here and only allow subclasses to override `#template` instead of a constant.

Reply:
Ok good call. I've updated the PR, thanks!